2008-06-09  Jakub Jelinek  <jakub@redhat.com>

	* omp.h.in (omp_nest_lock_t): Fix up for Linux multilibs.

--- libgomp/omp.h.in.jj	2008-06-09 13:34:05.000000000 +0200
+++ libgomp/omp.h.in	2008-06-09 13:34:48.000000000 +0200
@@ -42,8 +42,8 @@ typedef struct
 
 typedef struct
 {
-  unsigned char _x[@OMP_NEST_LOCK_SIZE@] 
-    __attribute__((__aligned__(@OMP_NEST_LOCK_ALIGN@)));
+  unsigned char _x[8 + sizeof (void *)] 
+    __attribute__((__aligned__(sizeof (void *))));
 } omp_nest_lock_t;
 #endif
 
